c_glib/test/test-array.rb

```diff
@@ -116,6 +116,18 @@ def test_to_s
     CONTENT
   end
 
+  def test_validation_success
+    array = build_int32_array([1, 2, 3, 4, 5])
+    assert_equal(true, array.validate)
```

Comment:
Could you use power-assert style for predicate?

```suggestion
    assert do
      array.validate
    end
```

Reply:
Fixed.